graylog2-server/src/main/java/org/graylog2/streams/StreamRouterEngine.java

```diff
@@ -180,7 +216,7 @@ public String getFingerprint() {
 
             for (final StreamRule streamRule : stream.getStreamRules()) {
                 try {
-                    final Rule rule = new Rule(stream, streamRule);
+                    final Rule rule = new Rule(stream, streamRule, null);
```

Comment:
What meaning would `sufficient = null` have in this case? There's also not a single check for `Rule#isSufficient() == null`.